SourceMods/src.share/shr_stream_mod.F90

```diff
@@ -0,0 +1,3322 @@
+! !MODULE: shr_stream_mod -- Data type and methods to manage input data streams.
```

Comment:
in the past tim has added a comment line at the top of any srcmods that gives the original pathname for this file.  it makes it easy to diff this file with the original source.  i think it's an inspired idea.